snuba/state/cache/redis/backend.py

```diff
@@ -297,9 +337,20 @@ def get_readthrough(
             return function()
 
         try:
-            return self.__get_readthrough(
-                key, function, record_cache_hit_type, timeout, timer
+            # set disable_lua_scripts to use the simple read-through cache without queueing.
+            sample_rate = get_config(
+                "read_through_cache.disable_lua_scripts_sample_rate", 0
             )
+            assert sample_rate is not None
```

Comment:
This should be an `if sample_rate is not None`. Otherwise someone could break queries simply by setting a bad value here (or having no value).

Reply:
@evanh above line sets the defualt value.  
sample_rate = get_config(
    "read_through_cache.disable_lua_scripts_sample_rate", 0
)

Reply:
Implemented fix.